README.md

```diff
@@ -34,38 +34,33 @@ Note that you have to change YOUR_API_KEY with your own API key!
 
 ## Usage
 
-### NavitiaSDKUX.init(config, success, failure)
+### NavitiaSDKUI.init(config, success, failure)
 
 | Parameters | Type | Required | Description | Example |
 | --- | --- |:---:| --- | --- |
 | config | Object | ✓ | Configuration | |
 | config.token | String | ✓ | Token navitia (generate a token on [navitia.io](https://www.navitia.io/))| 0de19ce5-e0eb-4524-a074-bda3c6894c19 |
+| config.mainColor | String | ✗ | To set the background and the journey's duration colors  | by default #2a968f |
+| config.originColor | String | ✗ | To set the origin icon's color  | by default #00b981 |
+| config.destinationColor | String | ✗ | To set the destination icon's color  | by default #b90054 |
```

Comment:
The "originColor" and "destinationColor" colors don't only affect the icon but also the roadmap departure and arrival blocs